flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/LegacyDataStreamQueryOperation.java

```diff
@@ -55,7 +55,7 @@
     private final boolean[] fieldNullables;
     private final FlinkStatistic statistic;
 
-    public DataStreamQueryOperation(
+    public LegacyDataStreamQueryOperation(
```

Comment:
This one is actually `Internal` and not `Legacy`. It is only used for testing. The other one is `Legacy`.